### DISCUSSION TITLE
2.0

### DISCUSSION THREAD

**Discussion by Unleashed2k** [yajra/laravel-datatables#3128] [Q&A]
Has this been tested to be fully compatible with the new datatables 2.0 update? 

https://cdn.datatables.net/2.0.0/

**Comment by yajra** [comment]
I haven't migrated my projects to 2.0 but the last time I tried, seems to work fine when I created a quick application. 

I am migrating my personal starter application to Laravel 11 (partially done) and hopefully DT 2.0. Will let you know how it works out. But I will support v2 as we go along.

**Comment by joriseduardo** [comment]
Does anyone know if there is official support for datatables js 2.0?

**Comment by yajra** [comment]
2.0 is supported and I'm using it on a current project. Thanks!
